.travis.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+if [[ -z $RIAK_FLAVOR ]]
+then
+    echo 'RIAK_FLAVOR must be set' 1>&2
+    exit 1
+fi
+
+if [[ $RIAK_DOWNLOAD_URL == 'docker' ]]
+then
+    docker run --name "$RIAK_FLAVOR" -d -p 8087:8087 -p 8098:8098 "basho/$RIAK_FLAVOR"
```

Comment:
Note that `$RIAK_FLAVOR` didn't used to be used at the end of this command (previously always `basho/riak-ts`). I assume that we want to use `basho/riak-kv` when that's the flavor.